src/main/java/com/icthh/xm/ms/entity/domain/listener/ProfileElasticSearchListener.java

```diff
@@ -18,12 +19,18 @@
 public class ProfileElasticSearchListener {
 
     private static XmEntitySearchRepository xmEntitySearchRepository;
+    private static XmEntityElasticRepository xmEntityElasticRepository;
```

Comment:
elsati it's details of implementation, please use search repository at this level